uvls/src/core/ast/transform.rs

```diff
@@ -1037,20 +1100,28 @@ fn visit_blk_decl(state: &mut VisitorState, parent: Symbol) {
             visit_group(state, parent, mode);
         }
         "cardinality" => {
-            check_langlvls(state, LanguageLevel::SAT(vec![LanguageLevelSAT::GroupCardinality]));
+            check_langlvls(
+                state,
+                LanguageLevel::SAT(vec![LanguageLevelSAT::GroupCardinality]),
+            );
             let card = opt_cardinality(state.node(), state).unwrap_or(Cardinality::Any);
             visit_group(state, parent, GroupMode::Cardinality(card));
         }
         _ => {
-            state.push_error(40, "expected a feature or group declaration");
+            if state.kind() == "constraint" && state.name(state.cursor().node()).contains("-") {
```

Comment:
It seems like this is the only semantic change in this file.    
Looks like you use some kind of Code formatter that changes a lot in formatting.  
While it makes sense to have a standard code style, it may be confusing for Review.  
Additionally, in my opinion many multi-line formats blow up the code unnecessarily.   
I'm ok with changes in spacing and commas or braces, but a call to some private language with a descriptive name like `check_langlvls` may not need to be spread over 4 lines (as it is above).  
However, this might be some part of the Rust Coding Guideline and therefore reasonable.  
Please check whatever you prefer here.

Reply:
I agree and I think we need a best practice for the project. Like everyone is using the same Code formatter or something.